_data/l10n.yml

```
@@ -151,6 +151,7 @@ it:
       sort_by_vitality: Più attivi
       sort_by_alphabetic: Alfabetico
       categories: Categorie
+      intendend_audience: Ambito di applicazione
```

```suggestion
      intended_audience: Ambito di applicazione
```
yetanothertypo? :rofl:

---

omg...